doc/kazoo/cluster-guide.md

```diff
@@ -1,9 +1,320 @@
+`sup -necallmgr ecallmgr_maintenance list_fs_nodes`
+
+### HAProxy config
+
+Edit `/etc/kazoo/haproxy/haproxy.cfg` on zone 100 Kazoo and Freeswitch server as follows.
+```
+global
+        log /dev/log local0 info
+        maxconn 4096
+        user haproxy
+        group daemon
+        stats socket    /var/run/haproxy/haproxy.sock mode 777
+
+defaults
+        log global
+        mode http
+        option httplog
+        option dontlognull
+	option log-health-checks
+        option redispatch
+        option httpchk GET /
+        option allbackups
+        option http-server-close
+        maxconn 2000
+        retries 3
+        timeout connect 6000ms
+        timeout client 12000ms
+        timeout server 12000ms
+        
+listen bigcouch-data 127.0.0.1:15984
+  balance roundrobin
+    server couch1.z100.somedomain.com 10.100.10.1:5984 check
+    server couch2.z100.somedomain.com 10.100.10.2:5984 check
+    server couch3.z100.somedomain.com 10.100.10.3:5984 check
+    server couch1.z200.somedomain.com 10.200.10.1:5984 check backup
+    server couch2.z200.somedomain.com 10.200.10.2:5984 check backup
+    server couch3.z200.somedomain.com 10.200.10.3:5984 check backup
+
+listen bigcouch-mgr 127.0.0.1:15986
+  balance roundrobin
+    server couch1.z100.somedomain.com 10.100.10.1:5986 check
+    server couch2.z100.somedomain.com 10.100.10.2:5986 check
+    server couch3.z100.somedomain.com 10.100.10.3:5986 check
+    server couch1.z200.somedomain.com 10.200.10.1:5986 check backup
+    server couch2.z200.somedomain.com 10.200.10.2:5986 check backup
+    server couch3.z200.somedomain.com 10.200.10.3:5986 check backup
+
+listen haproxy-stats 127.0.0.1:22002
+  mode http
+  stats uri /
+```
+### Kamailio Config
+
+Each Kamailio configuration at `/etc/kazoo/kamailio/local.cfg` needs to be configured with it's hostname, IP address, and all RabbitMQ servers.  The following config would be for the `ka1.z100` server.
+```
+## CHANGE "" TO YOUR SERVERS HOSTNAME
+#!substdef "!MY_HOSTNAME!kamailio1.z100.somedomain.com!g"
+
+## CHANGE "127.0.0.1" TO YOUR SERVERS IP ADDRESS
+##     Usually your public IP.  If you need
+##     to listen on addtional ports or IPs
+##     add them in "BINDINGS" at the bottom.
+#!substdef "!MY_IP_ADDRESS!10.100.50.1!g"
+
+## CHANGE "kazoo://guest:guest@127.0.0.1:5672" TO THE AMQP URL
+##     This should be the primary RabbitMQ server 
+##     in the zone that this server will service.
+#!substdef "!MY_AMQP_URL!kazoo://guest:guest@10.100.30.1:5672!g"
+
+## CHANGE "kazoo://guest:guest@127.0.0.1:5672" TO THE AMQP URL for the other zone.
+##     This uses the existing MY_AMQP_SECONDARY_URL variable defined in default.cfg
+##     Note the addition of the "zone=" part in the middle 
+#!substdef "!MY_AMQP_SECONDARY_URL!zone=z200;kazoo://guest:guest@10.200.30.1:5672!g"
+```
+
+### Kamailio Dispatcher
```

Comment:
I still cannot verify that.  On a brand new kamailio install with no `/etc/kazoo/kamailio/dbtext/dispatcher` file, starting kamailio creates /etc/kazoo/kamailio/db/kazoo.db but there is nothing in there.  I have to add the freeswitch IP's to the database manually.  

It don't see how it can grab that info from anywhere else anyways.  Only other place freeswitch is configured is in bigcouch ecallmgr document and it is the FQDN, not the IP.